packages/neuron-wallet/.env

```diff
@@ -110,3 +110,10 @@ TESTNET_NFT_DEP_INDEX=2
 TESTNET_NFT_DEP_TYPE=code
 TESTNET_NFT_SCRIPT_CODEHASH=0xb1837b5ad01a88558731953062d1f5cb547adf89ece01e8934a9f0aeed2d959f
 TESTNET_NFT_SCRIPT_HASH_TYPE=type
+
```

Comment:
These hard coded scripts can be updated
https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/models/system-script-info.ts#L8-L10